haskell/test/Test/Ligo/BaseDAO/Proposal.hs

```diff
@@ -135,6 +131,9 @@ test_BaseDAO_Proposal =
 
       , testScenario "correctly track freeze history" $ scenario $
           checkFreezeHistoryTracking (originateLigoDaoWithConfigDesc dynRecUnsafe)
+
+      , testScenario "can unstake tokens after flush is called" $ scenario $
```

Comment:
nitpick, but I suggest to change this to something like:
```suggestion
      , testScenario "tokens are unstaked correctly, only when possible" $ scenario $
```
simply because the scenario tests for incorrect cases and dropped proposals as well

Reply:
Make sense, I've updated this.